src/uct/tcp/tcp_cm.c

```diff
@@ -679,7 +677,11 @@ unsigned uct_tcp_cm_handle_conn_pkt(uct_tcp_ep_t **ep_p, void *pkt, uint32_t len
     case UCT_TCP_CM_CONN_REQ:
         /* Don't trace received CM packet here, because
          * EP doesn't contain the peer address */
-        ucs_assertv(length == sizeof(*cm_req_pkt), "ep=%p", *ep_p);
+        ucs_assertv((length ==
+                     (sizeof(*cm_req_pkt) +
+                      (ucs_derived_of((*ep_p)->super.super.iface,
+                                      uct_tcp_iface_t))->config.sockaddr_len)),
```

Comment:
use helper var for "uct_tcp_iface_iface_t *iface"

Reply:
this is used only in assert(). helper var becoming unused variable in non debug builds

Reply:
can use UCS_V_UNUSED